statsmodels/tsa/tests/test_stattools.py

```diff
@@ -349,6 +349,13 @@ def test_coint():
             r1 = res1[i][2]
             assert_allclose(r1, r2, rtol=0, atol=6e-7)
 
+    # use default autolag #4490
+    res1_0 = coint(y[:, 0], y[:, 1], trend='ct', maxlag=4)
+    assert_allclose(res1_0[2], res_egranger['ct'][0][1:], rtol=0, atol=6e-7)
+    # the following is just a regression test
```

Comment:
I'm assuming this is a regression test refers to the autolag kwarg being passed through?  If not, more explanation is needed.  If so, it might be better to combine this comment with the one on 352 for clarity.

Reply:
It's a regression test in the sense that the numbers are the current results and not from a verified source.
It will fail if either the lag selection changes, or the computation of the test statistic and p-values.

Reply:
OK.  I guess the GH chat will serve as clarification to future readers.

Reply:
It's our usual definition of a regression test. I don't see a need to make this more explicit.

Reply:
Yah, its not a problem.  I complain about enough around here; this comment doesn't merit membership in the pile.